cli.js

```diff
@@ -130,6 +131,7 @@ var api = new Api({
 	serial: cli.flags.serial,
 	require: arrify(cli.flags.require),
 	cacheEnabled: cli.flags.cache !== false,
+	disablePowerAssert: cli.flags.powerAssert === false,
```

Comment:
The CLI handler seems to handle `no-` well, to it sets `powerAssert` to false when usng `no-power-assert`. I think we should keep this.